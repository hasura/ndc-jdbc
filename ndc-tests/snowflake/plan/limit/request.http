POST /sql
content-type: application/json
x-hasura-ndc-version: 0.1.6

{
  "plan": {
    "type": "Limit",
    "input": {
      "type": "Project",
      "input": {
        "type": "Limit",
        "input": {
          "type": "From",
          "collection": "ARTIST",
          "columns": [
            "ARTISTID",
            "NAME"
          ]
        },
        "fetch": 2,
        "skip": 0
      },
      "exprs": [
        {
          "type": "Column",
          "index": 0
        },
        {
          "type": "Column",
          "index": 1
        }
      ]
    },
    "fetch": 2,
    "skip": 0
  }
}