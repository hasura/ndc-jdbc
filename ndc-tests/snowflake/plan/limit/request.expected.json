[
  {
    "rlp_col_0": 1,
    "rlp_col_1": "AC/DC"
  },
  {
    "rlp_col_0": 2,
    "rlp_col_1": "Accept"
  }
]